gen/aircraft.py

```diff
@@ -1210,7 +1210,7 @@ def build(self) -> MovingPoint:
         waypoint.alt_type = self.waypoint.alt_type
         waypoint.name = String(self.waypoint.name)
         tot = self.flight.flight_plan.tot_for_waypoint(self.waypoint)
-        if tot is not None:
+        if tot is not None and not self._viggen_client_tot():
```

Comment:
I think this should be moved into `set_waypoint_tot`. Pretty sure doing it here prevents the TOT from appearing in the kneeboard. We want to avoid setting it in the pydcs waypoint, but still want the kneeboard to be accurate.

Reply:
Makes sense, moved it down the chain in 91c037e - both the Viggen custom kneeboard and the liberation kneeboard look good with this.